api/ingress/v1alpha1/astartedefaultingress_webhook.go

```diff
@@ -104,9 +106,9 @@ func (r *AstarteDefaultIngress) validateAstarteDefaultIngress() error {
 			allErrors = append(allErrors, err)
 		}
 	}
-	if err := r.validateBrokerServiceType(); err != nil {
-		allErrors = append(allErrors, err)
-	}
+	// if err := r.validateBrokerServiceType(); err != nil {
+	// 	allErrors = append(allErrors, err)
+	// }
```

Comment:
Leftover? We're now handling this via kubebuilder tags

Reply:
Good catch. Removed.